summaries/2023-01-27.md

```diff
@@ -0,0 +1,37 @@
+# MOB Programming Session
+
+Date: 27-01-2023
+
+Topic: Unit test with Jest
+
+# Related Documentation
+
+> [Jest](https://jestjs.io/docs/getting-started)
+
+## Summary
+
+A unit test is a way of testing a unit - the smallest piece of code that can be logically isolated in a system. In most programming languages, that is a function, a subroutine, a method or property. The isolated part of the definition is important.
+
+Jest is an open source JavaScript unit testing framework, used by Facebook to test all JavaScript code including React applications. Jest is built on top of Jasmine. Unit testing of JavaScript code without jest framework needs Test runner such as Mocha or Jasmine, assertion library such as Chai, mocks, stubs, mocking library such as Sinon. But Jest is having in built all features available.
```

Comment:
`used by Facebook to test all JavaScript code including React applications.`

This is not relevant right